app/views/users/_tabs.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-xs-6">
+    <ul class="nav nav-tabs">
+      <li role="presentation" class="<%= "active" if params[:controller] == "users" %>"><a href="<%= users_path %>">Users</a></li>
```

Comment:
I think the same code exists for the navs, so I think that's reasonable to add now. I'll tackle this also tonight. How about `active_controller_class("users")`? Not sure what the best name would be... or perhaps `active_class controller: "users"`? I'm just thinking about how it checks against controller, but other active checks may check something else, so I'd like the method to be clear what it is doing without having to look at the method. I'm sort of leaning to the latter.

Thoughts?